src/components/BooksItem.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+
+import { removeBook } from '../redux/books.js/booksSlice';
 
 export default function BooksItem({ list }) {
```

Comment:
- [OPTIONAL] The way you defined the props type for the BooksItem component is incorrect and it produces this error in the console:

![image](https://github.com/techmoves/bookstores/assets/77106962/ac6ecd6b-3035-456b-b5c2-c794054158c3)

You can read this [article](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html) or [this article](https://www.freecodecamp.org/news/how-to-use-proptypes-in-react/) to understand the different prop types and how to shape up the correct prop types for each variable. 👍 😎